src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+.list {
+  list-style: none;
+  margin-left: 20px;
```

Comment:
remove margin-left from first element

Reply:
Thank you, i fixed all what u sayd me, but tests failed again, what i does wrong?